server/www/templates/file-upload/file-upload.html

```diff
@@ -1,2 +1,17 @@
-<input id="appearance-header-image" name="file" type="file" custom-on-change='uploadFile' ng-required="{{ required }}" />
+<!-- default input is hidden -->
+<input id="appearance-header-image" name="file" type="file" custom-on-change='uploadFile' ng-required="{{ required }}"/>
 
+<!-- "Add" and "Replace" buttons -->
```

Comment:
These buttons caused the custom-on-change='uploadFile' to be fired, is that correct @middle8media?